### PR TITLE
Promote qbits as itrex module

### PR DESCRIPTION
**Pull request by zhewang1-intc**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: intel_extension_for_transformers/transformers/llm/operator/csrc/CMakeLists.txt [hunk 1]
```diff
@@ -31,16 +31,20 @@ endif()
 find_package(Torch REQUIRED
 PATHS ${torch_path}
 NO_DEFAULT_PATH)
+find_package(PythonLibs 3 REQUIRED)
+execute_process(COMMAND python -c "import pybind11; print(pybind11.__path__[0]+\"/share/cmake/pybind11\")"
```
Comment (a32543254) [inline intel_extension_for_transformers/transformers/llm/operator/csrc/CMakeLists.txt:35]: why don't add pybind11 as third party ?  then we can simply use add_subdirectory(third_party/pybind11) 
to add this dependence

and this way seems not bind any pybind11 version

**Comment by a32543254** [comment]
how about we add a doc to intro this new api to costumers?

File: intel_extension_for_transformers/transformers/llm/operator/csrc/CMakeLists.txt (hunk 1 above)
Comment (zhewang1-intc) [reply intel_extension_for_transformers/transformers/llm/operator/csrc/CMakeLists.txt:35]: pybind11 gitmodule in runtime/third_party/ dir, if we set SKIP_RUNTIME=True, we will not fetch all gitmodules which will lead compile fail.

**Comment by zhewang1-intc** [comment]
> how about we add a doc to intro this new api to costumers?

sure

**Comment by VincyZhang** [comment]
pylint is fixed in this PR: https://github.com/intel/intel-extension-for-transformers/pull/1403

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by hshen14 — APPROVED** [review]

**Review by PenghuiCheng — APPROVED** [review]

**Pull request merged**